### DISCUSSION TITLE
홈설정 갤러리 및 그리드 화면 문제

### DISCUSSION THREAD

**Discussion by GOGOON1** [pronist/hELLO#103] [Q&A] [answered]
제가 스킨을 등록할때 올릴거 다 업로드해서 올렸는데도 불구하고 홈설정에 들어가서 갤러리나 그리드를 추가하여 설정하면 한줄로 나열되서 나오는데 무엇이 문제인지를 도무지 [모르겠습니다...ㅠ]
https://gotraffic.tistory.com

**Comment by pronist** [comment, marked as answer]
@GOGOON1 안녕하세요! 이 문제는 이미 #60 에서 보고된 이슈입니다. 아직 처리는 되지않았지만 추후 패치에서 처리를 할 예정입니다.

**Reply by GOGOON1** [reply]
답변해주셔서 감사합니다!
